tests/test.linter.js

```diff
@@ -114,31 +106,30 @@ describe('Linter', () => {
     expect(output.summary.warnings).toEqual(0);
   });
 
-  it('should collect an error when not an xpi/zip', () => {
+  it('should collect an error when not an xpi/zip', async () => {
     const addonLinter = new Linter({ _: ['tests/fixtures/not-a-zip.zip'] });
     // Stub print to prevent output.
     addonLinter.print = sinon.stub();
     expect(addonLinter.collector.errors.length).toEqual(0);
-    return addonLinter.scan()
-      .catch(() => {
-        expect(addonLinter.collector.errors.length).toEqual(1);
-        expect(addonLinter.collector.errors[0].code).toEqual(
-          messages.BAD_ZIPFILE.code);
-      });
+    try {
+      await addonLinter.scan();
```

Comment:
@hameleonka I'm wondering if we could use `await expect(addonLinter.scan()).rejects...` here instead of the `try...catch`, do you mind to give it a try? (e.g. it would also answer the question: "what is it supposed to be its rejected value? is it passing because it rejects and it passes the assertions or is it passing because it doesn't reject and no assertion are actually running?")